### PR TITLE
Dev zhi fixpack uwp

### PR DESCRIPTION
**Pull request by zhili1208**
https://github.com/NuGet/Home/issues/5379

pack layout files for UWP

### CODE REVIEW

**Comment by rohit21agrawal** [comment]
This will fail on every project that does not define that target. You need to ask the UWP project team to add these files to BuiltProjectOutptGroup target, this is the wrong way to go.

**Review by rohit21agrawal — CHANGES_REQUESTED** [review]
get an email thread going with UWP team on adding these layout files to a output group that pack supports.

File: src/NuGet.Core/NuGet.Build.Tasks.Pack/Pack.targets [hunk 1]
```diff
@@ -319,13 +319,14 @@ Copyright (c) .NET Foundation. All rights reserved.
...
-          DependsOnTargets="BuiltProjectOutputGroup;DocumentationProjectOutputGroup;SatelliteDllsProjectOutputGroup;_AddPriFileToPackBuildOutput;$(TargetsForTfmSpecificBuildOutput)"
+          DependsOnTargets="BuiltProjectOutputGroup;DocumentationProjectOutputGroup;SatelliteDllsProjectOutputGroup;_AddPriFileToPackBuildOutput;$(TargetsForTfmSpecificBuildOutput);PrepareLibraryLayout"
           Returns="@(BuildOutputInPackage)">
     <ItemGroup>
       <BuildOutputInPackage Include="@(SatelliteDllsProjectOutputGroupOutput);
                             @(BuiltProjectOutputGroupOutput);
                             @(DocumentationProjectOutputGroupOutput);
-                            @(_PathToPriFile)"/>
+                            @(_PathToPriFile);
+                            @(_LayoutFile)"/>
```
Comment (rohit21agrawal) [inline src/NuGet.Core/NuGet.Build.Tasks.Pack/Pack.targets:329]: we shouldn't take dependencies on items that start with _ as they are meant to be internal.

**Comment by commonsensesoftware** [comment]
@rohit21agrawal UWP is not open source to my knowledge. What is the _right_ way to engage with the team. It's not clear who to track down, even internally.

**Comment by rohit21agrawal** [comment]
@commonsensesoftware PM me and i'll send you some contacts.

**Comment by emgarten** [comment]
Please re-open if this is still active.

**Pull request closed without merging**